### ISSUE TITLE
DrawLineAa anti-aliased behavior is not as expected in a case

### ISSUE THREAD

**Issue by epsi1on** [reneschulte/WriteableBitmapEx#35]
Hi,
i have trouble with DrawLineAa method. drawn line is like below image:
it is drawn by DrawLineAA, but not anti aliased!

![line aa](https://user-images.githubusercontent.com/7839141/30309397-6b2b4c92-97a0-11e7-9e34-61f6739c1db1.png)
more detail:
![line aa2](https://user-images.githubusercontent.com/7839141/30309558-2b544b86-97a1-11e7-8653-7defedf66db5.png)

Do you know what is the reason of this? I'm using nuget package of WritableBitmapEx v1.5.1 on .NET 4.5.2 and windows 10
thanks

**Comment by brinkdinges** [comment]
I'm having this issue as well. Does the anti-aliasing assume a black background or something?

I could not get other other methods for drawing lines with a thickness to work, so I'm sticking with this method for now.

**Comment by mbenoni7** [comment]
Yeah, it appears to be a problem when the line is drawn onto transparent pixels. Filling the bitmap with a solid color first improves the appearance.

**Comment by reneschulte** [comment]
This would require significant time from my end which I don't have. If you can help out, a contribution is always welcome. Otherwise please use the workaround.

**Comment by epsi1on** [comment]
@reneschulte,
Hi, I'm not familiar with anti alias line drawing algorithms, can you please give some information or reference about used algorithm in the code?
I've traced this issue and made pull request #74 , but seems i broke the algorithm. any comment is welcome ! 
Thanks

**Comment by epsi1on** [comment]
Also do you accept Crypto Currency (Lite Coin) as donation? I do not have paypal but want to donate with LTC.
Thanks

**Comment by epsi1on** [comment]
guys I think we should do the job our self. hopefully will not break anything...
@brinkdinges  @mbenoni7 @EricOuellet2  do you have any experience in anti aliasing algorithms?
